bpf/include/linux/bpf.h

```diff
@@ -3449,7 +3456,9 @@ struct bpf_sock_addr {
 	__u32 msg_src_ip6[4];	/* Allows 1,2,4,8-byte read and 4,8-byte write.
 				 * Stored in network byte order.
 				 */
+#if 0 /* TODO */
```

Comment:
It's 300 lines apart from previous comment, maybe add “See comment in struct __sk_buff”?

Reply:
Still checking how we can work-around it to fix the issue w/o these workarounds.